<p>
This lesson will introduce you to the basics of gene expression analysis
using RNA-Seq (short for RNA sequencing). Due to the considerable
progress and constant decreasing costs of RNA-Seq, this technique has
became a standard technique in biology.

It is going to be fun and empowering! You will discover how total RNA
are converted to short sequences called “reads” that can in turn be used
to get insights into gene expression. Through careful experimental
design, these gene expression information can yield new research avenues
and answer crucial questions.

We will use mostly R and its companion RStudio to perform our RNA-Seq
analyses and visualisations.

Depending on the level of participants, the bioinformatic part might be
performed (QC of fastq files, genome alignment, counting, etc.). For
this, we will use a Docker image containing all necessary datasets and
softwares.
</p>
<p>
After completing this lesson, you will be able to:

<ul>
<li>Indicate the reasons for doing an RNA-Seq experiment.</li>
<li>Identify good practices when designing a RNA-Seq experiment.</li>
<li>Memorize the steps of a complete RNA-Seq experiment: from sequencing to analysis.</li>
<li>Assess the quality of RNA-seq sequencing data (“reads”) using the command-line
instructions in the cloud (Linux).</li>
<li>Align RNA-seq reads to a reference genome using a splice-aware aligner (e.g. STAR).</li>
<li>Generate a count matrix from the RNA-seq data alignment</li>
<li>Perform a QC of your experiment through Principal Component Analysis (PCA) and sample
clustering.</li>
<li>Execute a differential gene expression analysis using R and the DESeq2 package.</li>
<li>Be able to create key plots: volcano plot, heatmap and clustering of differentially
expressed genes.</li>
<li>Provide a biological interpretation to differentially expressed genes through ORA/GSEA
analyses and data integration.</li>
</ul>
</p>

